module/VuFind/src/VuFind/Role/DynamicRoleProvider.php

```diff
@@ -160,10 +160,11 @@ protected function getRolesForSettings($settings)
             $mode = 'ALL';
         }
 
-        // Extract permission setting:
+        // Extract permission and assertion setting:
```

Comment:
Would this be a little more accurate?

```suggestion
        // Extract permission setting and ignore assertion setting (it's processed in PermissionManagerFactory):
```